packages/analyticsdx-template-lint/test/unit/linter/autoInstall.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
`linter.test.ts` was getting pretty big, so I split it up by linted file type, so a lot of this `linter/*.test.ts` code isn't new.